### ISSUE TITLE
Use time format with seconds by default

### ISSUE THREAD

**Issue by ericgl** [myfreeer/7z-build-nsis#24]
Hi,
Thanks for releasing **7-Zip** with NSIS support.

I was wondering if you could modify one more thing:
By default, **7-Zip** displays the date and time in the **Modified/Created/Accessed** Date fields, with the time section showing only hours and minutes (HH:mm).
I can change it from the **View** menu (see screenshot I linked). However, it only persists in current session. 
Next time I open **7-Zip**, it defaults back to HH:mm only. But my proffered time format is **HH:mm:ss** .
Could you modify the code so that the the change is persistent?

Here is a screenshot:
https://imgur.com/a/r77Cg5P

Thanks in advance!
**Eric**

**Comment by myfreeer** [comment]
If this appies to official version, maybe it's better to be reported to the official tracker at https://sourceforge.net/p/sevenzip/discussion/45797/

**Comment by ericgl** [comment]
Yes, it also applies to the official version. But logically, I believe it should be a simple code fix...
